packages/spor-alert-react-native/src/ExpandableAlert.tsx

```diff
@@ -4,62 +4,55 @@ import {
 } from "@vygruppen/spor-icon-react-native";
 import React, { useState } from "react";
 import { Box } from "@vygruppen/spor-layout-react-native";
-import { Text } from "@vygruppen/spor-typography-react-native";
 import { Pressable } from "react-native";
 import { BaseAlert, ColorVariants } from "./BaseAlert";
+import { Text } from "@vygruppen/spor-typography-react-native";
 
 type ExpandableAlertProps = {
   children: React.ReactNode;
-  color: ColorVariants;
-  icon: JSX.Element;
+  colorScheme: ColorVariants;
+  leftIcon: JSX.Element;
   title: string;
+  defaultExpanded?: boolean;
   onToggle?: (isExpanded: boolean) => void;
 };
 
 export const ExpandableAlert = ({
   children,
-  color,
-  onToggle,
+  colorScheme,
   title,
```

Comment:
Kanskje vi burde bytte `title` til `heading` i det offentlige APIet også? Dvs i `ClosableAlert`, `ExpandableAlert` og `SimpleAlert` også?

Reply:
Enig. Endret Closable og Expandable, men Simple har ikke heading. Tenkte at children er enklere å forstå når det ikke er en overskrift? Eller burde man ha heading for helhetens skyld? 😊